databases/databases/databases_and_sql.md

```diff
@@ -147,7 +112,7 @@ The last nifty trick is if you want to only display a subset of your data.  In a
   HAVING posts_written >= 10;
 ~~~
 
-Try going back to [the W3 example](http://www.w3schools.com/sql/trysql.asp?filename=trysql_select_groupby) and joining the `Customers` and the `Orders` tables to get the number of orders in each country and adding the line `HAVING COUNT(*) > 10;` after `GROUP BY` (and delete the extra semicolon in the previous line).
+Try going back to [the W3 Schools' example](http://www.w3schools.com/sql/trysql.asp?filename=trysql_select_groupby) and joining the `Customers` and the `Orders` tables to get the number of orders in each country and adding the line `HAVING COUNT(*) > 10;` after `GROUP BY` (and delete the extra semicolon in the previous line).
```

Comment:
Assuming W3 School is singular, we would want `School's` here

Reply:
Nope, they are indeed called W3Schools. https://www.w3schools.com/about/default.asp But I might be daft, so check me on this 😆

Reply:
Seems a bit odd to me, but you're absolutely correct :+1: